src/package.json

```diff
@@ -3,14 +3,19 @@
   "version": "1.0.0",
   "description": "Monitor V2",
   "type": "module",
+  "engines": {
+    "node": "18.12.x",
+    "npm": "8.x"
+  },
   "scripts": {
     "prestart": "npm run build",
     "start": "node app.js",
     "dev": "nodemon app.js",
-    "build": "node esbuild & npm run build:png && npm run build:svg && npm run copy:root",
-    "build:png": "squoosh-cli --oxipng '{level: 2}' --quant '{numColors: 255}' -d ../dist/images ./client/images/*.png",
+    "build": "node esbuild & npm run copy:root & npm run copy:webp & npm run build:svg",
     "build:svg": "svgo -f ./client/images/ -o ../dist/images",
+    "copy:webp": "rsync -d --include='*.webp' ./client/images/ ../dist/images/",
     "copy:root": "rsync -d ./client/ ../dist/",
+    "convert:webp": "sh ./utils/webp.sh",
```

Comment:
This is meant to be run locally? I understand that a encoder is needed to run the script

Reply:
yes – if any .png files are added we'll want to run this to convert them to .webp.  It would obviously be me running this for the foreseeable future.

I think we need to start a readme for V2...

Reply:
I'm just wondering how much work it is to have it as a part of the CICD pipeline. 
We would have to add a step to install the cwebp...

Oh another thing I forgot to mention, let's not put `.sh` in util. I think it makes sense to have a `script` folder for all the scripts that are not a part of the application code

Reply:
@mansaj ok, I will start a `script` folder and add that file.

Having it in the CICD pipeline is tricky in ways that might take too long to explain in writing.  For .png files it made sense since there's a high likelihood of them being copied into the folder uncompressed.  But .webp arrive in the folder already compressed – this is probably better handled manually as needed.

Reply:
Sounds good